src/admin-views/widgets/list.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Admin View: List Widget
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/list.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @var Widget_Abstract $widget_obj   An instance with the widget abstract.
+ * @var array<array>    $admin_fields An array of admin fields to display in the widget form.
+ *
+ * @version TBD
+ */
+
+use \Tribe\Events\Views\V2\Widgets\Widget_Abstract;
+use Tribe__Utils__Array as Arr;
+
+if ( empty( $admin_fields ) ) {
+	return;
+}
+
+foreach ( $admin_fields as $id => $field ) {
+
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $id

<sub>hash: 9c1c7901fd11dc8c0db68625f2f5f0ef</sub>